whole-tale/girder_wholetale
# Tale importing and AinWT from Zenodo

This PR introduces Zenodo "integration" route that allows to import published Tales.

Note: Depends on https://github.com/whole-tale/gwvolman/pull/98

## How to test?
1. Create and publish a Tale to sandbox.zenodo.org. Make sure it includes a) registered dataset, b) some files in the workspace.
1. Remove Tale locally.
1. Open an incognito session
1. Go to published record and follow `Run this Tale on Whole Tale by clicking here.` in the record's description.
1. You should be asked to login and after a while you should end up on dashboard's run page for freshly imported Tale
1. (optionally) Make some modification and publish the Tale again. Confirm that it created new version, rather than a new deposition.
1. Remove the Tale
1. Navigate to swagger page, use `POST /tale/import` by providing only `url` to your Tale publish on zenodo (e.g. https://sandbox.zenodo.org/record/430905). Confirm that Tale was created properly and contains all data.
